packages/arb-rpc-node/arbostest/helper_test.go

```diff
@@ -196,29 +196,53 @@ func failIfError(t *testing.T, err error) {
 	}
 }
 
-func runAssertion(t *testing.T, inboxMessages []inbox.InboxMessage, logCount int, sendCount int) ([]value.Value, [][]byte, *snapshot.Snapshot, *protocol.ExecutionAssertion) {
+func runAssertion(t *testing.T, inboxMessages []inbox.InboxMessage, logCount int, sendCount int) ([]value.Value, [][]byte, *snapshot.Snapshot) {
 	t.Helper()
-	logs, sends, snap, assertion := runAssertionWithoutPrint(t, inboxMessages, logCount, sendCount)
-	testCase, err := inbox.TestVectorJSON(inboxMessages, assertion.Logs, assertion.Sends)
-	failIfError(t, err)
-	t.Log(string(testCase))
-	return logs, sends, snap, assertion
+	logs, sends, snap := runAssertionWithoutPrint(t, inboxMessages, logCount, sendCount)
+	//testCase, err := inbox.TestVectorJSON(inboxMessages, logs, sends)
+	//failIfError(t, err)
+	//t.Log(string(testCase))
```

Comment:
Should these lines just be deleted?

Reply:
I actually want to leave them in in case we want to re-enable them. Basically that produces output that can be used to recreate the test in an external environment (the aolog format). That could be used for example to run the test directly against the bare-metal AVM, but it adds a bunch of annoying log output even when we don't need it which is why I commented it out